src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java

```diff
@@ -72,15 +57,4 @@ public void testExploreWikisLink() {
     Assertion.assertEquals(driver.getCurrentUrl(), fandomUrlBuilder.getFandomUrl(EnvType.PROD) + "explore");
   }
 
-  @Test(groups = {"communityCentralLinkClickOnEnCommunityOpensEnCommunityCentral"})
-  public void testCommunityCentralLink() {
```

Comment:
@qaga why do we remove this test? and the one above?

Reply:
Because there is no Community Central link in global nav any longer, we replaced it with i18n hubs links, right?